paper-slider.html

```diff
@@ -474,7 +474,7 @@
       ready: function() {
         // issue polymer/polymer#1305
         this.async(function() {
-          this._updateKnob(this.value);
+          this._updateKnob(this.value, this.min, this.max);
         }, 1);
```

Comment:
I don't think we need to call `_updateKnob` here at all inside `ready`.  the [observer](https://github.com/PolymerElements/paper-slider/blob/01ad379495af47f8b77e1361fe014415001d3dff/paper-slider.html#L458) should be already taking care of calling `_updateKnob` at the right time with the right values.